tests/components/islamic_prayer_times/test_init.py

```diff
@@ -120,7 +120,10 @@ async def test_islamic_prayer_times_timestamp_format(hass: HomeAssistant) -> Non
         await hass.config_entries.async_setup(entry.entry_id)
         await hass.async_block_till_done()
 
-        assert hass.data[islamic_prayer_times.DOMAIN].data == PRAYER_TIMES_TIMESTAMPS
+        assert (
+            hass.data[islamic_prayer_times.DOMAIN][entry.entry_id].data
+            == PRAYER_TIMES_TIMESTAMPS
```

Comment:
Don't assert details like the coordinator or things in hass.data. Assert state in the state machine.